eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Data.hs

```diff
@@ -98,12 +121,7 @@ import qualified PlutusLedgerApi.V1 as Plutus
 -- It is imported from the Plutus package, but it needs a few additional
 -- instances to also work in the ledger.
 
-instance FromCBOR (Annotator Plutus.Data) where
-  fromCBOR = pure <$> fromPlainDecoder Cborg.decode
-
-instance ToCBOR Plutus.Data where
-  toCBOR = fromPlainEncoding . Cborg.encode
-
+-- TODO: Move to PlutusCore.Data module
```

Comment:
want to make an issues in that repo? https://github.com/input-output-hk/plutus/issues

Reply:
Sure, will do

Reply:
Done: https://github.com/input-output-hk/plutus/issues/4983